src/pymordemos/elliptic2.py

```diff
@@ -30,6 +36,8 @@ def elliptic2_demo(args):
     args['PROBLEM-NUMBER'] = int(args['PROBLEM-NUMBER'])
     assert 0 <= args['PROBLEM-NUMBER'] <= 1, ValueError('Invalid problem number.')
     args['N'] = int(args['N'])
+    norm = args['NORM']
+    norm = int(norm) if not (str(norm)[0]=='h' or str(norm)[0]=='l') else norm
```

Comment:
```suggestion
    norm = int(norm) if not norm.lower() in ('h1', 'l2') else norm.lower()
```

Otherwise, surprising things will happen when you specify 'h2' or 'l1' as norm parameter ..